src/clarity-angular/utils/expand/if-expanded.spec.ts

```diff
@@ -56,46 +56,46 @@ export default function(): void {
             });
 
             it("re-instantiates the content every time it is displayed", function () {
-                this.rowExpand.expanded = true;
+                this.expand.expanded = true;
                 this.fixture.detectChanges();
                 expect(this.testElement.textContent.trim()).toBe("1");
-                this.rowExpand.expanded = false;
+                this.expand.expanded = false;
                 this.fixture.detectChanges();
-                this.rowExpand.expanded = true;
+                this.expand.expanded = true;
                 this.fixture.detectChanges();
                 expect(this.testElement.textContent.trim()).toBe("2");
             });
         });
 
-        describe("Row interaction", function() {
+        describe("Row interaction", function () {
```

Comment:
Lots of leftover from Datagrid. You probably want to go over the `describe` and `it` strings in this file and rename them to something more generic.